app/mixins/resize-dropdown.js

```diff
@@ -0,0 +1,32 @@
+import Ember from 'ember';
+
+export default Ember.Mixin.create({
+  // This also exists as a initializer to deal with dropdowns that appear off the
+  // bottom of the page.
+  resizeDropdown: function(event, data) {
```

Comment:
Can `app/initializers/extend-bootstrap.js` use this instead of having a duplicate copy of this function?